### PR TITLE
fix(cli): Forward stdlib option to lsp and graindoc

### PR DESCRIPTION
**Pull request by ospencer**
Just need to specify that this needs to be forwarded.

### CODE REVIEW

**Review by marcusroberts — APPROVED** [review]

**Review by marcusroberts — APPROVED** [review]
I like this even better!

File: cli/bin/grain.js [hunk 1]
```diff
@@ -75,11 +75,18 @@ class ForwardOption extends program.Option {
   }
 }
 
-program.forwardOption = function (flags, description, parser, defaultValue) {
+// Adds .forwardOption to commands. Similar to Commander's native .option,
+// but will forward the flag to the underlying program.
+program.Command.prototype.forwardOption = function (
```
Comment (peblair) [inline cli/bin/grain.js:80]: Out of curiosity, why was this change needed?

**Review by peblair — APPROVED** [review]
One question, but looks good.

File: cli/bin/grain.js [hunk 2]
```diff
@@ -193,6 +200,12 @@ program
 program
   .command("lsp <file>")
   .description("check a grain file for LSP")
+  .forwardOption(
+    "-S, --stdlib <path>",
+    "override the standard libary with your own",
+    null,
+    stdlibPath
+  )
```
Comment (phated) [inline cli/bin/grain.js:208]: If these are specified separately from the main program, then we need to add others. such as `--output`. I'm not sure all of them that'll need to be duplicated

**Review by phated — CHANGES_REQUESTED** [review]
One (big?) thing

File: cli/bin/grain.js (hunk 1 above)
Comment (ospencer) [reply cli/bin/grain.js:80]: It's not needed to fix the issue, but it's needed to be able to `.forwardOption` on the subcommand. We just decided it was cleaner to be consistent around `.forwardOption` versus `.addOption(new ForwardOption())`!

File: cli/bin/grain.js (hunk 2 above)
Comment (ospencer) [reply cli/bin/grain.js:208]: Yup yup. Having a realization now. I'll make these inherit all of the compiler config for now until we get it truly separated.

**Comment by ospencer** [comment]
@phated should be all set.

File: cli/bin/grain.js [hunk 3]
```diff
@@ -194,7 +201,9 @@ program
   .command("lsp <file>")
   .description("check a grain file for LSP")
   .action(
-    wrapAction(function (file, options, program) {
+    wrapAction(function (file) {
```
Comment (phated) [inline cli/bin/grain.js:204]: I can't comment on the lines above the diff, but does this mean that the `run` command needs to be updated too?

Comment (phated) [reply cli/bin/grain.js:204]: Nevermind, it looks like that isn't using `wrapAction`

**Review by phated — APPROVED** [review]
I quickly tossed some commands I use at this and they seem to work 👍
